src/main/controllers/service/ChooseServiceController.ts

```diff
@@ -39,6 +42,6 @@ export class ChooseServiceController {
       return res.render('choose-service', data);
     }
 
-    return res.redirect('/services/' + req.body.chooseService + '/service-areas');
+    return res.redirect('/services/' + req.body.chooseService + '/service-areas/' + req.params.action as string);
```

Comment:
might be best to store in variable but not a big deal.

Reply:
Have changed `req.body.chooseService` and `req.params.action` to both be stored in variables now.